### PR TITLE
Misc fixes in Getting Started

### PR DESCRIPTION
**Pull request by akien-mga**
- Make links clickable
- Fix default structure of Godot project
- Fix plural in reference to macros from earlier versions
- Typo fixes

---

Some of the changes made might be up for discussion / a matter of taste, so feel free to tell me what you prefer to keep unchanged, and I'll amend the commit.

### CODE REVIEW

**Comment by akien-mga** [comment]
Another thing which I did not change here but noticed while reading: in https://github.com/godot-rust/book/blob/master/src/faq.md the `#[godot::methods]` attribute is used, while the (likely more up-to-date) Hello World tutorial uses only `#[methods]`. Is the former an earlier variant that should now be changed?

**Comment by karroffel** [comment]
Thanks a lot for the PR!

> the `#[godot::methods]` attribute is used, while the (likely more up-to-date) Hello World tutorial uses only `#[methods]`.

That's just a matter of style, all of these are equivalent in terms of Rust attributes
```rust
#[gdnative::methods]
impl MyType {...}
```
```rust
use gdnative::*;

#[methods]
impl MyType {...}
```

```rust
extern crate gdnative as godot;

#[godot::methods]
impl MyType {...}
```

That said, it should probably be made consistent, thanks for pointing that out!

File: src/getting-started/hello-world.md [hunk 1]
```diff
@@ -83,15 +82,15 @@ This declares an empty callback function, which is called when the library is lo
 godot_init!(init);
 ```
 
-These macros define the necessary C callbacks used by Godot. You only need *one* of these in the entire library. Note how the `init` function defined earlier is given to the `godot_nativescript_init` macro as a callback.
+This macro defines the necessary C callbacks used by Godot. You only need *one* of call to this macro in the entire library. Note how the `init` function defined earlier is given to the `godot_init!` macro as a callback.
```
Comment (karroffel) [inline src/getting-started/hello-world.md]:
```suggestion
This macro defines the necessary C callbacks used by Godot. You only need *one* invocation of this macro in the entire library. Note how the `init` function defined earlier is given to the `godot_init!` macro as a callback.
```

**Review by karroffel — CHANGES_REQUESTED** [review]
Nice catches! I made one small suggestion, other than that it should be good to go!

File: src/getting-started/hello-world.md (hunk 1 above)
Comment (akien-mga) [reply src/getting-started/hello-world.md]: Thanks, fixed.

**Review by karroffel — APPROVED** [review]
Looks fine to me, thanks a lot!

**Pull request merged**